x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/index.tsx

```diff
@@ -547,6 +570,16 @@ export const EndpointList = () => {
           rowProps={setTableRowProps}
         />
       );
+    } else if (canReadEndpointList && !canAccessFleet) {
```

Comment:
should this at all be based on fleet RBAC?

Reply:
sorry the test cases still needs to be updated, the modified policy empty state is what we are actually going to have when user has endpoint list read and fleet none